### PR TITLE
[T2A5][F11-A3] Irvin Lim

### PR DESCRIPTION
**Pull request by irvinlim**
Completed T2A2 through T2A5, combined into one PR.

### CODE REVIEW

**Comment by irvinlim** [comment]
Ready for review

**Comment by cmkumar87** [comment]
"Completed T2A2 through T2A5, combined into one PR." Okay, but why? Use a new git branch for each activity and issue a PR from each of those with commits relevant to only that activity. This is what you will do in a typical dev cycle.

File: test/expected.txt [hunk 1]
```diff
@@ -139,8 +139,9 @@
 || 0 persons found!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
```
Comment (cmkumar87) [inline test/expected.txt:141]: Good!

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -454,15 +455,31 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:455]: did you update header comment and the Usage variables (help) to say that find is now case insensitive?

**Pull request closed without merging**